### PR TITLE
fix(debian10): fix get-pip script

### PR DESCRIPTION
**Pull request by thalesmg**
```
0.220 ERROR: This script does not work on Python 3.7. The minimum supported Python version is 3.8. Please use https://bootstrap.pypa.io/pip/3.7/get-pip.py instead.
```

https://github.com/emqx/emqx-builder/actions/runs/9685773741/job/26727609446#step:11:14988

### CODE REVIEW

**Comment by id** [comment]
Debian 11 installs Python 3.9.2 by default.
May be we should just upgrade Python version here.

File: debian10/Dockerfile [hunk 1]
```diff
@@ -42,23 +42,23 @@ RUN apt-get update && apt-get install -y \
...
     && echo "_socket socketmodule.c" >> Modules/Setup.dist \
     && echo "_ssl _ssl.c -DUSE_SSL -I/usr/local/ssl/include -I/usr/local/ssl/include/openssl -L/usr/local/ssl/lib -lssl -lcrypto" >> Modules/Setup.dist \
-    && ./configure --prefix=/usr/local/python3.7.4 \
+    && ./configure --prefix=/usr/local/python3.9.2 \
     && make \
     && make install \
     && rm -rf /usr/bin/python3 /usr/bin/python \
-    && ln -s /usr/local/python3.7.4/bin/python3.7 /usr/bin/python3 \
-    && ln -s /usr/local/python3.7.4/bin/python3.7 /usr/bin/python
+    && ln -s /usr/local/python3.9.2/bin/python3.7 /usr/bin/python3 \
```
Comment (id) [inline debian10/Dockerfile:54]:
```suggestion
    && ln -s /usr/local/python3.9.2/bin/python3.9 /usr/bin/python3 \
```

File: debian10/Dockerfile [hunk 2]
```diff
@@ -42,23 +42,23 @@ RUN apt-get update && apt-get install -y \
...
+    && ln -s /usr/local/python3.9.2/bin/python3.7 /usr/bin/python
```
Comment (id) [inline debian10/Dockerfile:55]:
```suggestion
    && ln -s /usr/local/python3.9.2/bin/python3.9 /usr/bin/python
```

File: debian10/Dockerfile (hunk 1 above)
Comment (thalesmg) [reply debian10/Dockerfile:54]: :see_no_evil:

**Review by id — APPROVED** [review]

**Pull request merged**